gobblin-core/src/main/java/gobblin/instrumented/Instrumented.java

```diff
@@ -131,6 +138,85 @@ public static MetricContext getMetricContext(State state, Class<?> klazz, List<T
   }
 
   /**
+   * Copies a metric context and returns a new context with different name, and possibly different tags.
```

Comment:
Please mention that the copy does not include any metrics associated with the old context.